### PR TITLE
 feat:  Add options to the body of the request

### PR DESCRIPTION
**Pull request by yasiloghmani**
## Type

* ### Feature
  change body of the request, Add "options" to the body of the request
* ### Fix
  Fixes a bug
* ### Docs
  Documentation only changes
* ### Style
  Changes that **do not** affect the meaning of the code (white-space, formatting, missing semi-colons, etc)
* ### Refactor
  A code change that neither fixes a bug nor adds a feature
* ### Performance
  A code change that improves performance
* ### Test
  Adding missing or correcting existing tests
* ### Chore
  Changes to the build process or auxiliary tools and libraries such as documentation generation  

## Description

Add "options: {add_all_or_nothing:  boolean}}" to the body of the request

## Dependencies

*Other PRs or builds that this PR depends on.*

## Issues

*A list of issues closed by this PR.*

* Fixes #

## Notes

*Any additional notes.*

### CODE REVIEW

**Review by rostyk-kanafotskyy — APPROVED** [review]

**Review by shaunmaharaj — APPROVED** [review]

File: src/endpoints/cart.js [hunk 1]
```diff
@@ -67,12 +67,14 @@ class CartEndpoint extends BaseExtend {
...
-  BulkAdd(body, options) {
-    const payload = options ? { data: body, options } : body
+  BulkAdd(body) {
     return this.request.send(
       `${this.endpoint}/${this.cartId}/items`,
       'POST',
-      payload
+      {
+        data: body,
+        options: { add_all_or_nothing: false}
```
Comment (shaunmaharaj) [inline src/endpoints/cart.js:76]: Linter failing this check in build. Please add a space after the last curly brace
image

File: src/endpoints/cart.js [hunk 2]
```diff
@@ -67,12 +67,14 @@ class CartEndpoint extends BaseExtend {
     )
   }
 
-  BulkAdd(body, options) {
-    const payload = options ? { data: body, options } : body
+  BulkAdd(body) {
     return this.request.send(
       `${this.endpoint}/${this.cartId}/items`,
```
Comment (shaunmaharaj) [inline src/endpoints/cart.js:72]: @yasiloghmani the remaining linter errors are not resolved here and failing the build. Can you please take a look:
image

**Comment by shaunmaharaj** [comment]
@yasiloghmani test failure
image

**Comment by shaunmaharaj** [comment]
@yasiloghmani failed tests:
image

**Pull request merged**